nmpc_examples/pipeline/tests/test_pipeline_mhe.py

```diff
@@ -0,0 +1,57 @@
+import pyomo.common.unittest as unittest
+import pyomo.environ as pyo
+
+from nmpc_examples.pipeline.run_pipeline_mhe import run_mhe
+
+class TestPipelineMHE(unittest.TestCase):
+
+    def test_small_mhe_simulation(self):
+        simulation_data, estimate_data = run_mhe(
+            simulation_horizon=4.0,
+            sample_period=2.0,
+        )
```

Comment:
This test takes quite a while with the default estimator horizon (20 h, I think), so I shortened it. I end up getting the same answers either way. Does this make sense?

Reply:
Yes, it does make sense in this case and this is related to your side question.